src/viz_makie.jl

```diff
@@ -0,0 +1,238 @@
+
+"""
+    draw_system(sys; kwords...)
+
+Draw AtomsBase system with Makie.
+You need to load Makie backend for visualisation to work.
+
+`kwords` are passed to Makie to control drawing.
+
+See also `draw_system!` and `draw_trajectory`.
+
+# Kwords
+
+- `draw_cell=true`    : 
+- `cell_color=:cyan`  :
+- `scale=1.0`         : Scale atom sizes
+- `hide_axes=false`   : hide x-,y- and z-axes
+- any kword suported by `Makie.mesh`
+
+# Examples
+```julia
+using GLMakie
+using AtomsBuilder
+using AtomsView
+
+system = bulk(:Cu) * (4,4,4)
+draw_system(system)
+
+# draw cell
+draw_system(system; draw_cell=true, cell_color=:cyan, scale=1.0)
+```
+"""
+function draw_system(sys; kwords...)
+    fig = Figure(size = (1000, 1000))
+    draw_system!(fig[1,1], sys; kwords...)
+    return fig
+end
+
+"""
+    draw_system!(axis::Axis3, sys; kwords...)
+    draw_system!(fig::Union{GridPosition,GridSubposition}, system; kwords...)
+
+Draw AtomsBase to given `GridPosition` or `Axis3`.
+
+You can use this command to draw system on an existing figure.
+
+See also `draw_system`.
+
+# Kwords
+
+- `draw_cell=true`    : 
+- `cell_color=:cyan`  :
+- `scale=1.0`         : Scale atom sizes
+- `hide_axes=false`   : hide x-,y- and z-axes
+- any kword suported by `Makie.mesh`
+
+# Examples
+```julia
+using GLMakie
+using AtomsBuilder
+using AtomsView
+
+system = bulk(:Cu) * (4,4,4)
+
+fig = Figure()
+draw_system!(fig[1,1], system)
+
+
+# draw system without axes
+draw_system!(fig[1,2], system; hide_axes=true)
+
+display(fig)
+```
+"""
+function draw_system!(fig::Union{GridPosition,GridSubposition}, sys; kwords...)
+    axs = Axis3(fig, aspect = :data, perspectiveness = 0.5)
+    draw_system!(axs, sys; kwords...)
+    return fig
+end
+
+function draw_system!(axis::Axis3, sys; hide_axes=false, draw_cell=false, cell_color=:cyan, scale=1.0, kwords...)
+    @argcheck scale > 0
+    data = map( sys ) do at
+        r = position(at)
+        point = Point3f( ustrip.(u"Å", r) )
+        atom_number = atomic_number(at)
+        if 0 < atom_number < 110
```

Comment:
Why do you need this guard ? Again would a 3-argument `get` not cut it (to make use of future extensions of `PeriodicTable`